.travis.yml

```diff
@@ -35,14 +37,16 @@ install:
         fi
 
         if [ "$COMMAND" == "lorax-test" ]; then
+            npm rebuild
             node run build --with-coverage
```

Comment:
I would rather leave the build as it is. `node run build` will create an empty translations.json file instead of pulling from Zanata. This way, Zanata failures and bad translation data cannot cause the tests to fail.

There was an issue where translations.json was not created because I didn't create the directory first. This has been fixed.